docs/samples/browser-plugin-meetings/app.js

```diff
@@ -622,6 +622,11 @@ async function joinMeeting({withMedia, withDevice} = {withMedia: false, withDevi
       viewBreakouts();
     });
 
+    meeting.on('meeting:self:mutedByOthers', () => {
+      console.log('pkesari_received event to be muted by others');
```

Comment:
Please update the log comments everywhere added newly.

Reply:
These console logs will be removed. I have kept them for testing purpose

Reply:
Console logs are removed now